composer.json

```diff
@@ -17,6 +17,7 @@
   },
   "require-dev": {
     "api-clients/test-utilities": "^1.0",
+    "container-interop/container-interop": "^1.1",
```

Comment:
Yeah agree it is, but we might want to document it in the readme and add it to suggest notifying users when they install it in a project without one